Source/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ add_definitions(-D__STDC_CONSTANT_MACROS)
 
 macro(add_dolphin_library lib srcs libs)
 	add_library(${lib} STATIC ${srcs})
-	target_link_libraries(${lib} ${libs})
+	target_link_libraries(${lib} PUBLIC ${libs})
```

Comment:
So, it fails because SFML-Network is used by DolphinWX transitively through Core currently. I can fix that in another change.
But to be honest, I was thinking of rewriting and adding much more code to this macro at another point, so PUBLIC is the easiest for now.